src/pages/blog-post.jsx

```diff
@@ -1,11 +1,11 @@
-import React from "react";
-import moment from "moment";
-import ReactHtmlParser from "react-html-parser";
-import Stack from "../sdk/entry";
-import Layout from "../components/layout";
+import React from 'react';
+import moment from 'moment';
+import ReactHtmlParser from 'react-html-parser';
+import Stack from '../sdk/entry';
+import Layout from '../components/layout';
 
-import ArchiveRelative from "../components/archive-relative";
-import RenderComponents from "../components/render-components";
+import ArchiveRelative from '../components/archive-relative';
+import RenderComponents from '../components/render-components';
 
 class BlogPost extends React.Component {
```

Comment:
Please change the Class component to a functional component

Reply:
@Amitkanswal would address this separately